packages/amplify_core/doc/lib/auth.dart

```diff
@@ -483,6 +483,17 @@ Future<void> forgetSpecificDevice(AuthDevice myDevice) async {
 }
 // #enddocregion forget-specific-device
 
+// #docregion fetch-current-device
+Future<void> fetchCurrentUserDevice() async {
+  try {
+    final device = await Amplify.Auth.fetchCurrentDevice();
+    safePrint('Device: $device');
+  } on AuthException catch (e) {
+    safePrint('Get current device failed with error: $e');
```

Comment:
Should this say fetch current device?

Reply:
[fixed](https://github.com/aws-amplify/amplify-flutter/pull/5070/commits/045b5b71a1b0b9760a9bb0d8102aa136f682948e)